include/deal.II/lac/scalapack.h

```diff
@@ -40,57 +40,26 @@ DEAL_II_NAMESPACE_OPEN
  *
  * ScaLAPACK assumes that matrices are distributed according to the
  * block-cyclic decomposition scheme. An $M$ by $N$ matrix is first decomposed
- * into $MB$ by $NB$ blocks which are then uniformly distributed across
- * the 2D process grid $p*q \le Np$, where $p,q$ are grid dimensions and
- * $Np$ is the total number of processes.
+ * into $\lceil M / MB \rceil$ by $\lceil N / NB \rceil$ blocks which are then
+ * uniformly distributed across the 2D process grid $p*q \le Np$, where $p,q$
```

Comment:
no need for the `*` in the formula.

But this reads funny. There should be something between "...process grid" and the formula because the sentence could just end at "...process grid".